package.json

```diff
@@ -109,6 +109,7 @@
     "axios": "^1.6.7",
     "benchmark": "^2.1.4",
     "body-parser": "^1.20.2",
+    "c8": "^9.1.0",
```

Comment:
Does 8.x also work? 9.x version doesn't support Node 14 which we still support for the time being.

Reply:
Probably? Still unsure of this change though. Was trying to eliminate nyc due to vuln warnings, but tap still depends on it internally so it doesn't go away unless we update tap, but there are major changes in recent versions which make that non-trivial. I _think_ we can probably land this with the 8.x downgrade though to at least improve coverage speed, we just won't get the vuln fix we were hoping for. 😐